packages/lit-html/src/private-ssr-support.ts

```diff
@@ -57,6 +57,14 @@ export const _$LH = {
         return resolveOverrideFn(this, values);
       }
     },
+  patchDirectiveResolve: (
```

Comment:
Unsure if it's worth going a bit more verbose to communicate the condition
```suggestion
  patchDirectiveResolveIfNeeded: (
```